src/test.js

```diff
@@ -258,6 +258,7 @@ Test.prototype = {
 		}
 
 		emit( "testEnd", this.testReport.end( true ) );
+		this.testReport.slimAssertions();
```

Comment:
A comment to that effect (or at least a line break between the js-reporters-related and the old callback code) would make that much more clear. :smile: